### PR TITLE
Fix default menu actions, using Electron roles instead

### PR DESCRIPTION
**Pull request by Soreine**

### CODE REVIEW

File: src/DefaultEditMenu.js [hunk 1]
```diff
@@ -14,67 +14,20 @@ const DefaultEditMenu = React.createClass({
...
-                    onClick={this.onRedo}
-                    />
+                <MenuItem role="undo" />
+                <MenuItem role="redo" />
+                <MenuItem.Separator />
+                <MenuItem role="cut" />
+                <MenuItem role="copy" />
+                <MenuItem role="paste" />
+                <MenuItem role="delete" />
                 <MenuItem.Separator />
```
Comment (SamyPesse) [inline src/DefaultEditMenu.js]: Why is there a separator between `delete` and `selectall` ?

The menu for chrome is:

screen shot 2017-02-18 at 16 12 27

File: src/DefaultEditMenu.js [hunk 2]
```diff
@@ -14,67 +14,20 @@ const DefaultEditMenu = React.createClass({
...
-                    label="Redo"
-                    accelerator="Shift+CmdOrCtrl+Z"
-                    onClick={this.onRedo}
-                    />
+                <MenuItem role="undo" />
+                <MenuItem role="redo" />
+                <MenuItem.Separator />
+                <MenuItem role="cut" />
+                <MenuItem role="copy" />
+                <MenuItem role="paste" />
```
Comment (SamyPesse) [inline src/DefaultEditMenu.js]: Should we also add `pasteandmatchstyle` ? I'm not sure what it does exactly.

**Review by SamyPesse — APPROVED** [review]
Very good 👍 One small question / change request

**Review by SamyPesse — APPROVED** [review]

File: src/DefaultEditMenu.js (hunk 2 above)
Comment (Soreine) [reply src/DefaultEditMenu.js]: It pastes without formatting (using the format of the current text)

**Pull request merged**